### PR TITLE
adds scholarship data, demographic category constants, and tests.

### PR DESCRIPTION
**Pull request by Lorraine-Lyu**
All data are used for testing, if this branch is too big to review I can split it.

### CODE REVIEW

**Review by CarlosM10011 — APPROVED** [review]

**Review by MikolJiron — APPROVED** [review]

File: src/main/java/com/google/step/finscholar/data/DemographicCategories.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.data;
+
+/** The class with string constants about demographic categories. */ 
+public class DemographicCategories {
+
+    public static final String HISPANIC = "Hispanic";
```
Comment (hzimz) [inline src/main/java/com/google/step/finscholar/data/DemographicCategories.java]: Make this an enum instead.

File: src/main/java/com/google/step/finscholar/data/DemographicCategories.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public static final String WHITE = "White";
+    public static final String BLACK_OR_AFRICAN_AMERICAN = "Black or African American";
+    public static final String AMERICAN_INDIAN = "American Indian";
+    public static final String ALASKA_NATIVE = "Alaska Native";
+    public static final String ASIAN = "Asian";
+    public static final String PACIFIC_ISLANDER = "Native Hawaiian or Other Pacific Islander";
+  
+    public static final String INTERNATIONAL = "international student";
```
Comment (hzimz) [inline src/main/java/com/google/step/finscholar/data/DemographicCategories.java]: I'd break this into two enums:
Ethnicity
Gender

And these three into booleans.

File: src/test/java/com/google/step/finscholar/data/ScholarshipTest.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+   
+  @Before
+  public void setup() {
+    scholarshipData1 = ScholarshipSamples.scholarshipSampleOne;
+    scholarshipData2 = ScholarshipSamples.scholarshipSampleTwo;
+    customScholarship =  new Scholarship(TEST_NAME, TEST_ID, TEST_SCHOOLS, EMPTY_URL);
+  }
+
+  @Test
+  public void testGetName() {
```
Comment (hzimz) [inline src/test/java/com/google/step/finscholar/data/ScholarshipTest.java]: Test behaviors not methods. Getters probably don't need their own testing, setters can be tested by constructing one large scholarship object and making sure all the fields are set as expected. The behavior test could be "createsScholarshipForLowIncomeHispanicStudents"

**Review by hzimz — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/step/finscholar/data/DemographicCategories.java (hunk 2 above)
Comment (Lorraine-Lyu) [reply src/main/java/com/google/step/finscholar/data/DemographicCategories.java]: I moved ethnicities and gender into enums. And I think later all data will be directly retrieved from database, so I can just make uncommon fields to be string at this point.

**Review by CarlosM10011 — APPROVED** [review]

File: src/main/java/com/google/step/finscholar/data/Ethnicity.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+  private Ethnicity(String ethnicity) {
+    this.ethnicity = ethnicity;
+  }
+
+  /** @return The string associated with this constant. */
+  public String getValue() {
+    return this.ethnicity;
+  }
+
```
Comment (tinagrasso) [inline src/main/java/com/google/step/finscholar/data/Ethnicity.java]: nit: remove whitespace.

File: src/main/java/com/google/step/finscholar/data/Gender.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+  private Gender(String gender) {
+    this.gender = gender;
+  }
+
+  /** @return The string associated with this constant. */
+  public String getValue() {
+    return this.gender;
+  }
+
```
Comment (tinagrasso) [inline src/main/java/com/google/step/finscholar/data/Gender.java]: nit: remove whitespace.

**Review by tinagrasso — CHANGES_REQUESTED** [review]

**Review by tinagrasso — APPROVED** [review]

**Review by hzimz — APPROVED** [review]

**Pull request merged**